File: rad_anastasiya/bridge/src/com/universiry/bridge/shape/Square.java
package com.universiry.bridge.shape;

import com.universiry.bridge.color.IColor;

public class Square extends Rectangle {
    public Square(final double side, final IColor IColor) {
        super(side, side, IColor);
    }

    @Override
    public String toString() {
        return "Square{" +
                "side: " + first +
                ", color: " + IColor.name()
                + ", perimeter: " + perimeter()
                + ", square: " + square() +
                '}';
    }
}

File: kulikovskiy_georgiy/Bridge/Shape/Square.cs
namespace Bridge
{
  public class Square : Rectangle
  {
    public Square(double sideOne, IColor color)
        : base(sideOne, sideOne, color) { }

    public override string ToString()
    {
      return "Square: "
          + $"side: {SideOne}, "
          + $"perimeter: {Perimeter()}, "
          + $"area: {Area()}, "
          + $"color: {Color.GetColor()}";
    }
  }
}
